include/sdf/Box.hh

```diff
@@ -65,6 +68,12 @@ namespace sdf
     /// \return A reference to a gz::math::Boxd object.
     public: gz::math::Boxd &Shape();
 
+    /// \brief Calculate and return the Mass Matrix values for the Box
```

Comment:
Updated the docs for all the shapes in [16ce852](https://github.com/gazebosim/sdformat/pull/1299/commits/16ce852ad51294257cf3b836fe54a5f1ea79ae66)

> Ah, yeah, this is not ideal since it won't be obvious to the user that this will actually mutate the object. Can you add to the documentation that in the process of calculating the inertial, the material property of the object will be changed?

Made this change in the same commit as well